plugins/tencent/cos/cosargs.go

```diff
@@ -0,0 +1,24 @@
+package cos
+
+import (
+	"flag"
+	"github.com/7sDream/rikka/common/logger"
+)
+
+// flags for Tencent cloud cos sdk
+var ArgCosVersion = flag.String("bVersion", "", "Tencent cos sdk version")
```

Comment:
```suggestion
var ArgCosVersion = flag.String("tccosVer", "v4", "Tencent cos sdk version, v4(default) or v5")
```

为了保持兼容性，这里需要一个默认值。另外参数的说明文档也需要明确一些。

啊我想起来了，其他参数里的 b 的意思是 bucket，这里用 bVersion 不是特别合适，换成 `tccosVersion` 之类的吧……果然还是有点长？那  `tccosVer` 好了。